.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: node_js
 node_js:
 - 8
+before_install:
+- curl -s -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.0.1-amd64.deb
```

Comment:
You can also move these to the beginning of travis_deploy.sh, if ya want (right?)